.circleci/config.yml

```diff
@@ -89,7 +89,8 @@ jobs:
               $(lsb_release -cs) stable" | $SUDO tee /etc/apt/sources.list.d/docker.list > /dev/null
             $SUDO apt update
             $SUDO apt -y install docker-ce docker-ce-cli containerd.io
-      - browser-tools/install-browser-tools
+      - browser-tools/install-browser-tools:
+          replace-existing-chrome: true
```

Comment:
This ensures we always get the latest stable chrome version, which our browser testing relies on

Reply:
There may be some value in selecting an older chrome version to test against, so that we can be sure the code isn't relying on APIs/behaviors that we think are too new.

But in reality it's easiest to test against latest and we still benefit a lot from having this.

I only included this change because for some reason circle stopped installing the latest chrome by default, which broke our test